Add blog post about JEP 493 (enabled for Temurin 24)

jerboaa opened the pull request:
# Description of change
This adds a blog post entry for JDK 24 (Eclipse Temurin). Temurin enables JEP 493 and it's worth mentioning this as a blog with some frequently asked questions already answered.

Please review! Thank you very much in advance.

Thoughts?

karianna commented on content/blog/eclipse-temurin-jdk24-JEP493-enabled/index.md:
```diff
@@ -109,11 +110,11 @@ With that said, this is actually a `jpackage`
-for JDK 24 without service bindings that JDK 24.0.2 would use is (modulo some
-application specific modules):
+[JDK-8345185](https://bugs.openjdk.org/browse/JDK-8345185), has been fixed in
+JDK 24.0.2 (July 2025 release). Until then, it's suggestest to pass required
```
```suggestion
JDK 24.0.2 (July 2025 release). Until then, it's suggested to pass required
```

karianna commented:
Will need a date change (sorry I probably triggered that by adding an extra commit).

jerboaa commented:
> Will need a date change (sorry I probably triggered that by adding an extra commit).

No problem. Thanks for the review. I've cleaned up the commits and squashed them along the way.

jerboaa commented:
Preview is here:
https://deploy-preview-3391--eclipsefdn-adoptium.netlify.app/blog/2025/02/eclipse-temurin-jdk24-JEP493-enabled/

smlambert reviewed (approved):
LGTM, suspect that the overzealous checker may complain again about the date... EDIT: looks like it is ready to merge.

sxa reviewed (approved):
You're the expert in this so no issues with the content. My only comment would be that we could just say "JDK 24" here for simplicity instead of explicitly 0.0+36 (particularly since we implemented this a few versions back in the early access builds) but it's not something I'd insist on.

jerboaa commented:
> You're the expert in this so no issues with the content. My only comment would be that we could just say "JDK 24" here for simplicity instead of explicitly 0.0+36 (particularly since we implemented this a few versions back in the early access builds) but it's not something I'd insist on.

Thanks! `JDK 24 GA` is `24.0.0+36` (most likely). There is also `24.0.1` and `24.0.2` coming all of which would be included in "JDK 24" (?) and I wanted to be precise there.

jerboaa commented:
Thanks for the reviews! I'll update to the latest main branch and set the date for today. If all checks then pass, I'll merge.
